OrbitLinuxTracing/TracerThread.cpp

```diff
@@ -1016,20 +1016,27 @@ void TracerThread::ProcessDeferredEvents() {
   pthread_setname_np(pthread_self(), "Proc.Def.Events");
   bool should_exit = false;
   while (!should_exit) {
+    ORBIT_SCOPE("ProcessDeferredEvents iteration");
     // When "should_exit" becomes true, we know that we have stopped generating
     // deferred events. The last iteration will consume all remaining events.
     should_exit = stop_deferred_thread_;
     std::vector<std::unique_ptr<PerfEvent>> events = ConsumeDeferredEvents();
     if (events.empty()) {
       // TODO: use a wait/notify mechanism instead of check/sleep.
+      ORBIT_SCOPE("Sleep");
```

Comment:
In other places you add extra braces to clarify the exact scope, here you don't. I think either would be fine, but should be consistent.

Reply:
Braces are not needed because this covers the entire scope, unlike the other case.

Reply:
Misunderstood the other case, makes sense.